modules/olm-filtering-fbc.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+ifdef::openshift-origin[]
+:registry-image: quay.io/operator-framework/opm:latest
+endif::[]
+ifndef::openshift-origin[]
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v{product-version}
+endif::[]
+
+:_content-type: PROCEDURE
+[id="olm-filtering-fbc_{context}"]
+= Filtering a file-based catalog image
+
+You can use the `opm` CLI to filter, or prune, a catalog image that uses the file-based catalog format. By extracting and updating the contents of an existing catalog image, you can remove one or more Operator packages from the catalog, and then rebuild the image as an updated version.
+
+[NOTE]
+====
+Alternatively, if you already have a catalog image on a mirror registry, you can use the oc-mirror CLI plugin to automatically prune any removed images from an updated source version of that catalog image while mirroring it to the target registry.
+
+For more information about the oc-mirror plugin and this use case, see the "Keeping your mirror registry content updated" section, and specifically the "Pruning images" subsection, of "Mirroring images for a disconnected installation using the oc-mirror plugin".
+====
+
+.Prerequisites
+
+* `opm` CLI.
+* `podman` version 1.9.3+.
+* A file-based catalog image.
+* A catalog directory structure recently initialized on your workstation related to this catalog. If you do not have one, create the directory and generate the Dockerfile by following the "Initialize the catalog" step from the "Creating a file-based catalog image" procedure.
+
+.Procedure
+
+. Extract the contents of the catalog image in YAML format to an `index.yaml` file in your catalog directory:
++
+[source,terminal]
+----
+$ opm render <registry>/<namespace>/<catalog_image_name>:<tag> \
+    -o yaml > <catalog_dir>/index.yaml
+----
++
+[NOTE]
+====
+Alternatively, you can use the `-o json` flag to output in JSON format.
+====
+
+. Modify the contents of the resulting `index.yaml` file to remove one or more Operator packages.
++
+[IMPORTANT]
+====
+After a bundle has been published in a catalog, assume that one of your users has installed it. Therefore, avoid stranding users that have that version installed by making sure that all previously published bundles in a catalog have an update path to the current or newer channel head.
```

Comment:
```suggestion
After a bundle has been published in a catalog, assume that one of your users has installed it. Ensure that all previously published bundles in a catalog have an update path to the current or newer channel head to avoid stranding users that have that version installed .
```
This phrasing is a little clearer to me. WDYT?